.github/workflows/upload-idf-component.yml

```diff
@@ -0,0 +1,19 @@
+name: Push components to https://components.espressif.com
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  upload_components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Why is master used an not the latest version?

Reply:
Nice catch, `v2` will be a better choice,

@Ouss4  Could you please take a look, I made a PR to github-actions to update the readme: https://github.com/espressif/github-actions/pull/27